### PR TITLE
[1515] Add "powered by Google" text 

### PR DESCRIPTION
**Pull request by stanica**

### CODE REVIEW

File: cornfield/app.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ var mongoose = require('mongoose'),
     }),
     UserModel = mongoose.model( 'User', User );
 
-CONFIG.session.store = new MongoStore({ db: "test" });
+//CONFIG.session.store = new MongoStore({ db: "test" });
```
Comment (humphd) [inline cornfield/app.js]: If this is intentional, it needs an explaining comment, and a link to a bug where we're fixing it.

File: editors/googlemap-editor.css [hunk 2]
```diff
@@ -29,3 +29,10 @@ p#message {
 .error {
   border: 2px inset #FF0000;
 }
+#powered {
+
```
Comment (humphd) [inline editors/googlemap-editor.css]: Remove this extra line.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/ac9158ba2b60a82/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/8cace87b2c6299f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/ac9158ba2b60a82/output.txt

Total script time: 0.22 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/8cace87b2c6299f/output.txt

Total script time: 0.62 mins

Server is running on http://pr493.butyr.org/templates/test.html

**Pull request merged**